### DISCUSSION TITLE
restAPI file upload

### DISCUSSION THREAD

**Discussion by LWJ-SARA** [mattieserver/netbox-topology-views#555] [Q&A]
When I was testing with postman, I wanted to upload a csv file with a post request to add the IP address, but when I sent the request, it always returned {
    "address": [
        "This field is required."
    ]
}
In fact, I already have this field in my file, and I try to upload a txt file that contains json data, and the same problem occurs.
![111](https://github.com/user-attachments/assets/01b7685e-8727-461c-abcc-6aa7ac3f3dab)

**Comment by mattieserver** [comment]
This is not related to this plugin as we don't set adresses?

**Reply by LWJ-SARA** [reply]
Sorry, I sent the question to the wrong community.
